### PR TITLE
helpButton: Added help button with tooltip

### PR DESCRIPTION
**Pull request by carolinepar**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import './App.css';
 import { Container } from 'semantic-ui-react';
 import Map from './component/map/Map.js';
```
Comment (Jaega) [inline src/App.js]: Remember to change this path as well

Comment (carolinepar) [reply src/App.js]: Yeah I'm doing it right now, thanks!

**Review by Jaega — APPROVED** [review]
Looks good!

**Review by liruochen1998 — APPROVED** [review]
LGTM

**Comment by liruochen1998** [comment]
hey @carolinepar , just realized when I was debugging -- in react I think it complains (showing up warnings) for using keyword "class" instead of "className". I think we should fix it some point, as deploying will fail. Could you please change this really quick when you have time? I just restored this branch.

**Pull request merged**